sno/working_copy/postgis.py

```diff
@@ -166,61 +129,53 @@ def _sno_table(self, sno_table):
         return self._table_identifier(f"_sno_{sno_table}")
 
     def _table_identifier(self, table):
-        return Identifier(self.schema, table)
+        # TODO - use sqlalchemy not strings
+        return f'"{self.schema}"."{table}"'
 
     @contextlib.contextmanager
     def session(self, bulk=0):
         """
-        Context manager for PostgreSQL DB sessions, yields a connection object inside a transaction
+        Context manager for GeoPackage DB sessions, yields a connection object inside a transaction
 
         Calling again yields the _same_ connection, the transaction/etc only happen in the outer one.
         """
         L = logging.getLogger(f"{self.__class__.__qualname__}.session")
 
-        if hasattr(self, "_db"):
+        if hasattr(self, "_session"):
             # inner - reuse
-            L.debug("session: existing...")
-            yield self._db
-            L.debug("session: existing/done")
+            L.debug(f"session: existing...")
+            yield self._session
+            L.debug(f"session: existing/done")
+
         else:
-            L.debug("session: new...")
-            try:
-                self._db = psycopg2.connect(self.dburl, cursor_factory=DictCursor)
-            except Exception as e:
-                raise NotFound(str(e), exit_code=NO_WORKING_COPY)
-            self._configure_output(self._db)
-            self._register_geometry_type(self._db)
+            L.debug(f"session: new...")
 
             try:
-                yield self._db
-            except:  # noqa
-                self._db.rollback()
+                # TODO - use "with" for sqlalchemy transaction.
+                self._session = self.sessionmaker()
+                self._configure_output(self._session)
+                # self._register_geometry_type(self._session)
```

Comment:
```suggestion
```

Feels like these are/should be done in `sno.sqlalchemy.postgis_engine()`?

Reply:
Yes - dead code again.